src/test/java/com/sparta/eng82/interfaces/pages/navpages/trainer/addpages/AddGroupPage.java

```diff
@@ -4,19 +4,19 @@
 
 public interface AddGroupPage {
 
-    AddGroupPage selectStream(String streamName);
```

Comment:
Need to return AddGroupPage for chaining methods together. Will commit for now but expecting changes tomorrow morning.